test/Aqua.jl

```diff
@@ -2,5 +2,9 @@
     # The unbound_args test fails on methods with signature like foo(::Type{Tuple{Vararg{V}}}) where V
```

Comment:
Unrelated but this is a real unbound_args. `V` is unbound in `Tuple{Vararg{V}}` because `()` is technically a subtype - in which case `V` has no definition.

Correct signature with bounded `V` would be `Tuple{V,Vararg{V}}`